project/Frontend.scala

```diff
@@ -22,6 +22,7 @@ object Frontend extends Build with Prototypes {
       awsSns,
       awsSts,
       awsSqs,
+      cats,
```

Comment:
I would prefer not to introduce any new dependency unless we cannot do without them.